ClientApp/src/app/components/result-home/reportgenerator/reportgenerator.component.ts

```diff
@@ -171,24 +174,19 @@ export class ReportgeneratorComponent implements OnInit {
 
   GeneraIndicadores(year: number) {
 
-    for (let i = 1; i < this.indicatorGroups.length + 1; i++) {
-
-      this.service.getIndicatorGroup(i).subscribe(g => {
-        g.indicators.forEach(indicator => {
-          this.serviceIndicator.getIndicatorYearRegistries(indicator.indicatorID, this.selectedYear).subscribe(j => {
-            this.indicators.push(j);
-
-          });
-        });
-
-      });
-
+    for (let i = 0; i < this.indicatorGroups.length; i++) { 
```

Comment:
This change is not synced. Rationale states that it was a change intended for another branch. In this case, this case, both branches should be synced and emitted as a single pull request.